README.md

```diff
@@ -1,8 +1,89 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Vishay Singh . Jarvis Consulting
 
+Hello there! My name is Vishay Singh and I graduated from the University of Toronto with an Honours 
+Bachelor's Degree of Science, majoring in Computer Science and French Studies. Since a young age, I've
+been fascinated by what technology can do to solve real world problems, which is exactly what I want 
+to do in the software industry. In fact, learning new things is a passion of mine, so software provides 
+the perfect avenue to continually challenge myself. In my spare time, I'm either building computers, 
+playing video games, or learning guitar.
+
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git, Python, IDEs (IntelliJ, Visual Studio Code, Eclipse, PyCharm, etc.), REST API, Docker, DAO design pattern, CRUD, JDBC, Regular Expressions, MVC architecture, Spring/SpringBoot, JUnit, Lambda Expressions, Stream API
```

Comment:
Some of the more generic, highly-specific, or non-technical elements don't need to be listed here. E.g. IDEs, CRUD, RegEx, Lambda/Stream, MVC. These are either generally expected or included within other things you've listed.